packages/cli/lib/projectStructure.js

```diff
@@ -102,13 +104,14 @@ async function findProjectComponents(projectSourceDir) {
     if (Object.values(CONFIG_FILES).includes(base)) {
       const parsedAppConfig = loadConfigFile(projectFile);
 
-      if (parsedAppConfig && parsedAppConfig.name) {
+      if (parsedAppConfig) {
         const isLegacy = getIsLegacyApp(parsedAppConfig, dir);
+        const isHublTheme = base === CONFIG_FILES[COMPONENT_TYPES.hublTheme];
 
         components.push({
           type: getTypeFromConfigFile(base),
           config: parsedAppConfig,
-          runnable: !isLegacy,
+          runnable: !isLegacy && !isHublTheme,
           path: dir,
         });
       }
```

Comment:
@brandenrodgers @camden11 to confirm I'm not making a mistake in adding like this. Running the server through `hs project dev` is going to be a bit bigger chunk of work I'd like to avoid for the moment but I'm using this util to fetch components and then filter to just the theme components to select from - marking as not runnable here to prevent it from getting picked up by `hs project dev`, and then we have a check for the name property that _seems_ unnecessary & is filtering `theme.json`s because they don't have a `name` field

Reply:
This seems like it should be safe! tbh I can't remember why we added in the `name` check, but it's probably not necessary. There's a bunch of validation that we could be doing here to make sure the JSON files are formatted correctly and contain the required fields, but that work can all be done separately from this. You can keep what you have here 👍 

Also, I'm noticing the `parsedAppConfig` doesn't really make sense as a variable name anymore since now it can include themes. Feel free to update it to be something more generic if you'd like to.